core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -81,17 +99,29 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
     new File(subDir, filename)
   }
 
-  def getFile(blockId: BlockId): File = getFile(blockId.name)
```

Comment:
Hi, @LantaoJin . Why does this PR need to remove this function signature? Can we keep this function signature to minimize this PR's surface?